### PR TITLE
Add Drop Database and Drop Table support

### PR DESCRIPTION
**Pull request by travisturner**

### CODE REVIEW

File: dax/computer/service/computer.go [hunk 1]
```diff
@@ -123,27 +119,21 @@ type CommandConfig struct {
...
-		ssSvc = snapshotter.New(snapshotter.Config{
-			DataDir: cfg.SnapshotterConfig.DataDir,
-			Logger:  cfg.Logger,
-		})
+	var ssSvc computer.SnapshotService
+	if cfg.ComputerConfig.SnapshotterDir != "" {
+		ssSvc = snapshotter.New(cfg.ComputerConfig.SnapshotterDir, cfg.Logger)
+	} else {
+		ssSvc = computer.NewNopSnapshotterService()
+		cfg.Logger.Warnf("No snapshotter configured.")
```
Comment (jaffee) [inline dax/computer/service/computer.go:136]: if you want a no-op you should have to explicitly ask for it so we don't have folks accidentally running w/o snapshot/writelog and not knowing it.

...and by folks I mean us

File: dax/test/dax/dax_test.go [hunk 2]
```diff
@@ -507,6 +514,134 @@ func TestDAXIntegration(t *testing.T) {
...
+			numSlash := strings.Count(name, "/")
+			for i := 0; i < numSlash; i++ {
+				fmt.Print(" ")
+			}
+			fmt.Printf("%s\n", name)
+			return nil
+		})
+		if err != nil {
+			t.Fatalf("walking dir: %v", err)
+		}
```
Comment (jaffee) [inline dax/test/dax/dax_test.go:643]: finish this test... not actually checking anything. also need to clean up the duplicated walk func

**Review by jaffee — CHANGES_REQUESTED** [review]
the requested changes are in the comments

File: dax/computer/service/computer.go (hunk 1 above)
Comment (travisturner) [reply dax/computer/service/computer.go]: I changed this to accept `"NULL"` or error on `""`

File: dax/test/dax/dax_test.go (hunk 2 above)
Comment (travisturner) [reply dax/test/dax/dax_test.go]: just added some checks to see if the directories are empty or not

File: dax/computer/service/computer.go (hunk 1 above)
Comment (travisturner) [reply dax/computer/service/computer.go]: changed it to case-insensitive "off"

**Review by jaffee — APPROVED** [review]

**Pull request merged**